### PR TITLE
docs: document ckb-app-config

### PR DESCRIPTION
**Pull request by doitian**

### CODE REVIEW

File: util/app-config/src/configs/db.rs [hunk 1]
```diff
@@ -2,21 +2,32 @@ use serde::{Deserialize, Serialize};
...
-    /// TODO(doc): @doitian
+    /// Provide an options file to tune RocksDB for your workload and your system configuration.
+    ///
+    /// More details can be found in [the official tuning guide](https://github.com/facebook/rocksdb/wiki/RocksDB-Tuning-Guide).
     pub options_file: Option<PathBuf>,
 }
 
 impl Config {
-    /// TODO(doc): @doitian
+    /// Canonicalizes pathes in the config options.
```
Comment (yangby-cryptape) [inline util/app-config/src/configs/db.rs:25]:
```suggestion
    /// Canonicalizes paths in the config options.
```

File: util/app-config/src/cli.rs [hunk 2]
```diff
@@ -136,7 +136,9 @@ fn basic_app<'b>() -> App<'static, 'b> {
         .subcommand(migrate())
 }
 
-/// TODO(doc): @doitian
+/// Parse the command line argumets by supplying the version information.
```
Comment (yangby-cryptape) [inline util/app-config/src/cli.rs:139]:
```suggestion
/// Parse the command line arguments by supplying the version information.
```

File: util/app-config/src/args.rs [hunk 3]
```diff
@@ -6,157 +6,165 @@ use ckb_types::packed::Byte32;
...
-    /// TODO(doc): @doitian
+    /// Specifies the ending block number. The default is the tip block in the database.
     pub to: Option<u64>,
 }
 
-/// TODO(doc): @doitian
+/// Parsed command line arguments for `ckb init`.
 pub struct InitArgs {
-    /// TODO(doc): @doitian
+    /// Whether to promopt user inputs interactively.
```
Comment (yangby-cryptape) [inline util/app-config/src/args.rs:89]:
```suggestion
    /// Whether to prompt user inputs interactively.
```

**Review by yangby-cryptape — CHANGES_REQUESTED** [review]

**Review by yangby-cryptape — DISMISSED** [review]

**Review by zhangsoledad — APPROVED** [review]

**Comment by doitian** [comment]
Resolved conflicts with #2501

**Review by quake — APPROVED** [review]

**Comment by doitian** [comment]
Resolved conflicts

**Review by quake — APPROVED** [review]

**Review by yangby-cryptape — APPROVED** [review]

**Comment by doitian** [comment]
bors r=quake,yangby-cryptape

**Pull request merged**